packages/language-picker/src/language-picker.tsx

```diff
@@ -77,27 +113,96 @@ const LanguagePicker = ( { onSelectLanguage, languages, languageGroups }: Props
 			};
 
 			return (
-				<div key={ languageGroup.id }>
-					<Button onClick={ onClick } className="language-picker-component__language-group">
-						<span className={ isSelected ? 'is-selected' : '' }>{ languageGroup.name() }</span>
-					</Button>
-				</div>
+				<Button
+					key={ languageGroup.id }
+					onClick={ onClick }
+					className="language-picker-component__language-group"
+				>
+					<span className={ isSelected ? 'is-selected' : '' }>{ languageGroup.name() }</span>
+				</Button>
 			);
 		} );
 	};
 
+	const shouldDisplayRegions = ! search;
+
+	const languagesToRender = search
+		? getSearchedLanguages( languages, search, localizedLanguageNames )
+		: getFilteredLanguages();
+
+	const selectControlOptions = languageGroups.map( ( lg ) => ( { key: lg.id, name: lg.name() } ) );
+
 	return (
-		<div className="language-picker-component">
-			<div className="language-picker-component__regions-label">
-				{ __( 'regions', __i18n_text_domain__ ) }
-			</div>
-			<div className="language-picker-component__content">
-				<div className="language-picker-component__category-filters">
-					{ renderCategoryButtons() }
+		<Flex className="language-picker-component" align="left">
+			<FlexBlock className="language-picker-component__heading">
+				<Flex justify="space-between" align="left">
+					<FlexItem className="language-picker-component__title wp-brand-font">
+						{ headingTitle || __( 'Select a language', __i18n_text_domain__ ) }
```

Comment:
In some instances we use a textdomain, and in some we don't. What is the rationale behind that?

Reply:
It should be being used for all instances. Places that aren't are a mistake I believe (except for `createLanguageGroups` where the passed in `__` will have a text domain configured I believe... perhaps that is incorrect?).

Reply:
> It should be being used for all instances. 
Gotcha. That's aligned to pbAok1-1Ao-p2 so we should be good.

> `createLanguageGroups` where the passed in `__` will have a text domain configured I believe

I think that's not the case, it doesn't have the textdomain configured, but it doesn't need to have a textdomain, because where we pass `__` is used only in Calypso context, where we don't need to specify a textdomain. If the package is used externally, then we can just pass a bound `__()` that specifies a textdomain, which isn't ideal, but will do the job.

Reply:
Hmmm... In that case, we might as well pass `__i18n_text_domain__` in `createLanguageGroups` right? The signature should allow us to regardless and it doesn't cost anything right? Or is that incorrect?

Reply:
> we might as well pass `__i18n_text_domain__` in createLanguageGroups

I'm assuming yes, but I'm not 100% sure what you mean, could you please demonstrate it with an example?